install.sh

```diff
@@ -279,60 +271,58 @@ for i in $(seq 0 $((${#urls[@]} - 1))); do
   tarfile=$(basename "${url}")
   name="${tarfile%%-*}"   # extract string before first '-'
   rest="${tarfile#*-}"    # extract string after first '-'
-  version="${rest%%-chromeos*}"
-                          # extract string between first '-' and "-chromeos"
+  version="${rest%%-chromeos*}" # extract string between first '-' and "-chromeos"
 
   download_check "${name}" "${url}" "${tarfile}" "${sha256}"
   extract_install "${name}" "${tarfile}"
   update_device_json "${name}" "${version}"
 done
 
 echo_info "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
-echo -e "${GRAY}"
+
+# Work around https://github.com/chromebrew/chromebrew/issues/3305.
+sudo ldconfig &> /dev/null || true
+echo_out "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
 ln -sfv "../lib/crew/bin/crew" "${CREW_PREFIX}/bin/"
-echo -e "${RESET}"
 
-echo_info "Setup and synchronize local package repo..."
-echo -e "${GRAY}"
+echo_out "Set up and synchronize local package repo..."
 
-# Remove old git config directories if they exist.
-find "${CREW_LIB_PATH}" -mindepth 1 -delete
+# Set LD_LIBRARY_PATH so crew doesn't break on i686 and
+# the mandb postinstall doesn't fail.
+echo "export LD_LIBRARY_PATH=$CREW_PREFIX/lib$LIB_SUFFIX" >> $CREW_PREFIX/etc/env.d/path
+export LD_LIBRARY_PATH=$CREW_PREFIX/lib$LIB_SUFFIX
 
-curl -L --progress-bar https://github.com/"${OWNER}"/"${REPO}"/tarball/"${BRANCH}" | tar -xz --strip-components=1 -C "${CREW_LIB_PATH}"
+# Add the CREW_PREFIX bin and musl bin directories to PATH.
+echo "export PATH=$CREW_PREFIX/bin:$CREW_PREFIX/sbin:$CREW_PREFIX/share/musl/bin:$PATH" >> $CREW_PREFIX/etc/env.d/path
+export PATH=$CREW_PREFIX/bin:$CREW_PREFIX/share/musl/bin:$PATH
 
-# Set LD_LIBRARY_PATH so crew doesn't break on i686 and
-# the mandb install doesn't fail.
-export LD_LIBRARY_PATH="${CREW_PREFIX}/lib${LIB_SUFFIX}"
+echo "export CREW_PREFIX=${CREW_PREFIX}" >> "${CREW_PREFIX}/etc/env.d/profile"
 
-# Since we just downloaded the package repo, just update package
-# compatibility information.
+# Since we downloaded the package repo, just update package compatibility information.
 crew update compatible
 
 echo_info "Installing core Chromebrew packages...\n"
+# We need these to install core.
+yes | crew install rsync
+yes | crew install pixz
 yes | crew install core
 
 echo_info "\nRunning Bootstrap package postinstall scripts...\n"
-crew postinstall "${BOOTSTRAP_PACKAGES}"
+crew install "${BOOTSTRAP_PACKAGES}"
```

Comment:
Why the change?  The message says we're running postinstall scripts.

Reply:
`postinstall` gave a nasty red error stating that the packages weren't installed and refused to do the postinstall, although the script did continue. `install` runs the postinstall scripts and doesn't give any errors.

Reply:
Better wording?
```
echo_info "\nInstallng bootstrap packages...\n"
```

Reply:
But we are just running the postinstall scripts here, just via a slightly different method.

Reply:
You just mentioned there was an error because the packages weren't installed.  I'm confused.

Reply:
`crew postinstall` threw an error which stated the packages weren't installed, even though they were. `crew install` doesn't throw that error, and still runs the postinstall scripts.

Reply:
![image](https://github.com/chromebrew/chromebrew/assets/64618338/4345d2e4-8c2e-4b12-94db-8f24e84cd218)
(The postinstall error in question)

Reply:
Then the packages need installed.  What are your thoughts about this @satmandu?

Reply:
Never mind, it's a formatting issue:
![image](https://github.com/chromebrew/chromebrew/assets/64618338/8af4627e-f737-4f3e-81ca-0f3ad5fa399d)

Reply:
This needs to be:
```
echo_info "\nRunning Bootstrap package postinstall scripts...\n"
# shellcheck disable=SC2086
# Do NOT quote BOOTSTRAP_PACKAGES, otherwise crew thinks the list of
# packages passed to crew postinstall is a single package.
crew postinstall ${BOOTSTRAP_PACKAGES}
```